spark/components/masthead/vanilla/masthead.stories.js

```diff
@@ -1,3 +1,703 @@
+      </p>
+      <p class="sprk-u-mbm">
+        Amet sociis conubia pharetra dolor pharetra, arcu integer! Aliquam eget pellentesque ad etiam vivamus nostra. Nostra iaculis suscipit sed inceptos enim blandit convallis, convallis fringilla. Semper ultricies aliquam congue sed ipsum. Fames amet etiam accumsan urna fermentum, netus habitant ut dui est. Ipsum conubia ullamcorper consectetur ut congue placerat tempus, rutrum ac laoreet pretium aptent. Augue vehicula mollis vitae mi quis turpis commodo venenatis varius. Odio ullamcorper leo scelerisque ipsum fusce luctus per fusce tortor? Ligula sagittis himenaeos faucibus commodo. Montes cum, sociis.
+      </p>
+      <p class="sprk-u-mbm">
+        Cubilia congue curabitur aliquet accumsan turpis nulla. Gravida maecenas, augue diam felis auctor platea. Urna dictumst elementum per lobortis cum euismod ullamcorper. Eget mattis commodo integer volutpat potenti. Mollis nisl urna et, justo facilisis volutpat quis nascetur ipsum? Ac est nec dui torquent vulputate volutpat a eget suscipit dictum. Etiam tempor, curae; proin potenti nunc posuere primis tellus dui eget. Eget lectus mauris mollis imperdiet purus in himenaeos torquent elit curabitur. Cras erat libero eros ultricies placerat rhoncus nec rutrum, at mi. Luctus etiam lobortis taciti scelerisque. Tellus, fringilla nulla odio class phasellus. Aliquet praesent fringilla.
+      </p>
+      <p class="sprk-u-mbm">
+        Sollicitudin mollis consequat ligula et sed curabitur tempor. Nullam lobortis porttitor lectus scelerisque lacinia ridiculus? Elementum penatibus tristique est ornare? Imperdiet sit ultrices vehicula augue adipiscing primis. Duis, quis interdum dignissim consequat class ultrices mi. Molestie suspendisse pulvinar massa nec velit posuere venenatis ad. Amet dui ridiculus volutpat habitasse. Ut, fringilla neque fusce faucibus! Mi velit cras ante, in lobortis sit blandit ligula cras sodales donec suscipit. Nec commodo aptent euismod sodales quis tellus ipsum habitasse! Donec.
+      </p>
+    </div>
+  `
+};
+
+defaultMasthead.story = {
+  name: 'Default',
+}
+
+export const extended = () => {
+  useEffect(() => {
+    dropdowns();
+    toggle();
+    masthead();
+
+    // Our masthead.js applies load event to `window` but doesn't work in storybook iframes
+    // This useEffect runs this code when component mounts
+    const isMenuVisibleSB = isElementVisible('.sprk-c-Masthead__menu');
+    toggleScrollEvent(isMenuVisibleSB);
+  }, []);
+  return `
+    <header class="sprk-c-Masthead sprk-o-Stack" role="banner" idstring="masthead-2" data-sprk-masthead="">
```

Comment:
```suggestion
    <header class="sprk-c-Masthead sprk-o-Stack" role="banner" idstring="masthead-2" data-sprk-masthead>
```
Our code doesn't need this to be set to an empty string. Just data-sprk-masthead is all we need.